AlfrescoApp/App Delegate/AppDelegate.m

```diff
@@ -108,11 +108,7 @@ - (BOOL)application:(UIApplication *)application didFinishLaunchingWithOptions:(
             [hockeyManager.authenticator authenticateInstallation];
         }
         
-        // Flurry Analytics
-        if (FLURRY_API_KEY.length > 0)
-        {
-            [[AnalyticsManager sharedManager] startAnalytics];
-        }
```

Comment:
If we're removing Flurry (needs Product Management approval) then all references to Flurry should be removed (API KEY, etc.)